### DISCUSSION TITLE
Shell Navigation

### DISCUSSION THREAD

**Discussion by stevewasielewski** [Dreamescaper/BlazorBindings.Maui#134] [Q&A]
Try to use the following code to get a tab bar:

public App()
{
    AppHost = MobileBlazorBindingsHost.CreateDefaultBuilder()
        .ConfigureServices((hostContext, services) =>
        {
            // Register app-specific services
            //services.AddSingleton<AppState>();
            services.AddSingleton<ShellNavigationManager>();
        })
        .Build();

    MainPage = new ContentPage();
    AppHost.AddComponent<AppShell>(parent: this);
}
Am not sure if am missing something.  I have an AppShell.razor with the tabs setup.

Thanks Steve

**Comment by Dreamescaper** [comment]
It seems like you're using MobileBlazorBindings package (which is unsupported now).
You need this one instead: https://www.nuget.org/packages/BlazorBindings.Maui/.
Here's the example for Program class:
https://github.com/Dreamescaper/BlazorBindings.Maui/blob/main/samples/BlazorBindingsXaminals/MauiProgram.cs
